src/plugins/c/README.md

```diff
@@ -1,5 +1,6 @@
 - infos = Information about the c plugin is in keys below
 - infos/author = Markus Raab <elektra@libelektra.org>
+- infos/maintainer = Florian Lindner <florian.lindner@student.tuwien.ac.at>
```

Comment:
For these to compile we need to add the corresponding REGEXes to LibAddMacros.cmake

Reply:
I've merged the updated cmake macros into master. Please rebase.